### ISSUE TITLE
The minimum version required for Language and Country filtering is not made clear to mobile experiments owners

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/experimenter#7402]
**[Prerequisites]:**

* Have a mobile experiment draft Ready for Launch.

**[Steps to reproduce]:**

1. Open the page from prerequisites on the “Audience” section.
1. Select the “No Version” option for “Min Version”.
1. Select at least an option from the “Languages” or “Countries“ field.
1. Click “Save and Continue”.
1. Click the “Audience” link from “Missing details in: Audience”.
1. Observe the message displayed.

**[Expected result]:**

* The message displayed helps users who want to launch an experiment on all the versions available for “Languages”/”Countries” filtering choose the necessary “Min Version”.

**[Actual result]:**

* The “Languages/Countries are not supported for this version.” Validation Error message displayed is too vague. People who do not already know what the minimum supported versions are would end up having to try to guess the minimum version needed.

**[Notes]:**

* Attached is a recording of the issue.

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/EXP-2511)
┆Attachments: <a href="https://mozilla-hub.atlassian.net/rest/api/2/attachment/content/25078">min version language.gif</a>

**Comment by data-sync-user** [comment]
➤ Yashika Khurana commented:

Maria Heres it is fixed now, thank you

**Comment by data-sync-user** [comment]
➤ Maria Heres commented:

I have verified that the validation error message has been modified to “Language targeting is not supported for this application below version 101.!“. The change is also available for the Country message. The versions displayed are “101.” for iOS applications and “102.” for Android applications.

Tested using Firefox Release 101.0.1 (Build ID: 20220608170832) on Windows 10.
